src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -117,14 +129,27 @@ def compute_reference(frames: np.ndarray,
             frame[:] = shift_frame(frame=frame, dy=dy, dx=dx)
```

Comment:
I wonder if it would be better if we just build our own `shift_frame` or `shift_frames` which does not use `np.roll` at all, making it impossible to wrap. You could give the option to pad with nans in it. This would abstract a bit of the shifting logic into a function.

Reply:
Maybe? reusing roll with a pad was the simplest thing I could think of doing before averaging the image to create the reference to prevent wrapping.